deploy/tests/test.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'html/proofer'
+proofer = HTML::Proofer.new("./_site", {
+  :directory_index_file => "index.html",
+  :url_ignore => ["/dashboard","/feed/","18f@gsa.gov"],
+  :disable_external => true,
+  :href_swap => {"%20" => " "}
+})
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.